File: Examples/basics.py
# Commenting

# Talk about indenting and colons
def main():
    sep = '='*10

    # Println

    print(sep, 0, sep)

    print('Hello World')  # No need for ;

    # Print

    print(sep, 1, sep)

    print('Hello', end='')
    print('World')

    # Change end in python

    print(sep, 2, sep)

    print('Hello World', end='!\n')

    # Seperator

    # Default end='\n', sep=' '

    print(sep, 3, sep)

    print('Hello', 'World')

    # Change Seperator

    print(sep, 4, sep)

    print('Hello', 'World', sep='$$$')

    # Division

    print(sep, 5, sep)

    print(5 // 2)

    print(5 % 2)

    print(5 / 2)

    # Boolean types

    print(sep, 6, sep)

    print(True)
    print(False)
    print(None)  # Python null

    # Boolean statements

    print(sep, 7, sep)

    print(True and True)
    print(True and False)
    print(False and False)

    print(True or True)
    print(True or False)
    print(False or False)

    print(not True)
    print(not False)


main()

File: Examples/Basics.java
package PythonWorkshop.Examples;

// Commenting

class Basics {
    public static void main(String[] args) {
        String sep = "";
        for (int i = 0; i < 10; i++)
            sep += "=";

        // Println

        System.out.println(sep + 0 + sep);

        System.out.println("Hello World");

        // Print

        System.out.println(sep + 1 + sep);

        System.out.print("Hello");
        System.out.println("World");

        // Division

        System.out.println(sep + 5 + sep);

        System.out.println(5 / 2);

        System.out.println(5 % 2);

        System.out.println(5.0 / 2.0);

        // Boolean types

        System.out.println(sep + 6 + sep);

        System.out.println(true);
        System.out.println(false);

        // Boolean statements

        System.out.println(sep + 7 + sep);

        System.out.println(true && true);
        System.out.println(true && false);
        System.out.println(false && false);

        System.out.println(true || true);
        System.out.println(true || false);
        System.out.println(false || false);

        System.out.println(!true);
        System.out.println(!false);

    }
}